jwst/jump/jump_step.py

```diff
@@ -26,24 +26,24 @@ class JumpStep(Step):
         after_jump_flag_time1 = float(default=0) # 1st flag groups after jump groups within specified time
         after_jump_flag_dn2 = float(default=0) # 2nd flag groups after jump above DN threshold
         after_jump_flag_time2 = float(default=0) # 2nd flag groups after jump groups within specified time
+        expand_large_events = boolean(default=False) # Turns on Snowball detector for NIR detectors
         min_sat_area = float(default=1.0) # minimum required area for the central saturation of snowballs
         min_jump_area = float(default=5.0) # minimum area to trigger large events processing
         expand_factor = float(default=2.0) # The expansion factor for the enclosing circles or ellipses
         use_ellipses = boolean(default=False) # deprecated
         sat_required_snowball = boolean(default=True) # Require the center of snowballs to be saturated
         min_sat_radius_extend = float(default=2.5) # The min radius of the sat core to trigger the extension of the core
-        sat_expand = integer(default=2) Number of pixels to add to the radius of the saturated core of snowballs
-        expand_large_events = boolean(default=False) # Turns on Snowball detector for NIR detectors
-        find_showers = boolean(default=False) Turn on shower flagging for MIRI
-        edge_size = integer(default=25) # Size of region on the edges of NIR detectors where a sat core is not required
+        sat_expand = integer(default=2) Number of pixels to add to the radius of the saturated core of snowballs       
```

Comment:
Need to have a `#` preceding all the comment fields, otherwise they don't get parsed properly and show up when folks do a param list. There are several instances of this, starting here and continuing below.